roles/default/kiali-deploy/defaults/main.yml

```
@@ -93,7 +93,9 @@ kiali_defaults:
       discovery_auto_threshold: 10
       discovery_enabled: "auto"
       enabled: true
-      is_core_component: false
+      component_status:
+        is_core: false
+        health_check_url: ""
```

all peer settings should be sorted alphabetically. "h" before "i" :)

```suggestion
        health_check_url: ""
        is_core: false
```